cpmonitor/views.py

```diff
@@ -15,6 +15,37 @@
 from .models import City, ExecutionStatus, Task
 
 
+def _get_children(city, node=None):
+    statuses = {s.value: s for s in ExecutionStatus}
+
+    if not node:
+        children = Task.get_root_nodes()
+    else:
+        children = node.get_children()
+
+    groups = children.filter(city=city, numchild__gt=0)
+    tasks = children.filter(city=city, numchild=0)
+    for group in groups:
+        subtasks = group.get_descendants().filter(numchild=0)
+        total = len(subtasks)
+        group.statuses = {}
```

Comment:
The field somewhat expresses overall progress. However, it does not hold the progress by status. It holds the percentage of tasks in a given status.
Maybe `group.task_proportions` ? add `..._by_status`, if you like, but I am not the biggest fan of overly verbose variable names (because they are long) as you might have noticed ;-)